experimental/segmenter_lstm/src/structs.rs

```diff
@@ -14,7 +15,7 @@ use std::collections::HashMap;
 /// `mat2` - `mat4`: the matrices associated with forward LSTM layer (embedding to hunits, hunits to hunits, and bias respectively)
 /// `mat5` - `mat7`: the matrices associated with backward LSTM layer (embedding to hunits, hunits to hunits, and bias respectively)
 /// `mat8` - `mat9`: the matrices associated with output layer (weight and bias term respectiely)
-#[derive(PartialEq, Debug, Serialize, Deserialize)]
+#[derive(PartialEq, Debug, Serialize, Deserialize, Yokeable, Clone)]
 pub struct LstmData {
```

Comment:
We should be able to migrate this to ZeroVec.

However, we use non-zero-copy types from the `ndarray` crate here.  These were not necessarily intended to be in the permanent data model of the LSTM; additional benchmarking should be performed to evaluate whether we could eliminate the `ndarray` dependency.

Could you experiment with the different options here please?